app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="forms-section">
+  <h2>Log in</h2>
+
+  <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+    <div class="field">
+      <%= f.label :email %><br />
+      <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :password %><br />
+      <%= f.password_field :password, autocomplete: "current-password" %>
+    </div>
+
+    <% if devise_mapping.rememberable? %>
+      <div class="field">
+        <%= f.check_box :remember_me %>
+        <%= f.label :remember_me %>
+      </div>
+    <% end %>
+
+    <div class="actions">
+      <%= f.submit "Log in" %>
+    </div>
+  <% end %>
+
+  <%= render "devise/shared/links" %>
+</div>
```

Comment:
- [ ] As in the instructions, this view is supposed to be modified to match the wireframe below:
  
  ![image](https://user-images.githubusercontent.com/45002654/151014298-c62afeca-d564-4339-88dd-d89ba1ccdd14.png)
  
  Currently it is like this 👇🏿:  
  
  ![image](https://user-images.githubusercontent.com/45002654/151014406-ef618a23-e34f-4835-9e23-f03612bbd80c.png)
  
  As you make the change, remember to configure `devise` to only accept username and password for log_in as in the wireframe. Please consider working on this: